06 piškvorky/piskvorky1D.py

```diff
@@ -0,0 +1,77 @@
+from random import randrange
+import time
+
+# TICK-TACK-TOE
+
+
+def score(area):  # fce get list(area) and return result in accordance with area
+
+    # draw
+    if '-' not in area:
+        print(' "!" ')
+        quit()
+    # players won
+    elif (3 * player_symbol) in area:
+        print(' "X" ')
+        quit()
+    # pc won
+    elif (3 * pc_symbol) in area:
+        print('"O"')
+        quit()
+    # next play
+    else:
+        print('"-"')
+
+
+def move(area, position, symbol):  # put symbol into area, get area, position 1 - 19, symbol
+
+    area = list(area)
+    area[position - 1] = symbol
+    area = ''.join(area)
+    return area
+
+
+def player_move(area):  # get input(position) from player, test input and return area with symbol
+
+    position = input('Kam umístit X? Vyber políčko 1 - 20. Tvá volba: ')
+    while True:
+
+        position = str(position)
```

Comment:
Tu pretypovávaš na string, pričom ono to string už je by default. Aj keď by užívateľ zadal číslo, input funkcia z toho urobí  string.

Reply:
Tomuto sa vyvaruješ práve tým, že input vložíš na začiatok while cyklu a pri identifikácii chyby sa k nemu vždy vrátiš na začiatok pomocou continue :) T.j. po tom ako sa spýtaš znova tak zasa pôjdu všetky kontroly od znova a máš na začiatku vždy rovnaký vstup (string).